virtualizarr/tests/test_manifests/test_store.py

```diff
@@ -55,71 +97,75 @@ def manifest_store(filepath, array_v3_metadata):
     manifest_group = ManifestGroup(
         {"foo": manifest_array, "bar": manifest_array}, attributes={"Zarr": "Hooray!"}
     )
-    return ManifestStore(
-        stores={"file://": obs.store.LocalStore()}, manifest_group=manifest_group
-    )
+    return ManifestStore(stores={"s3://": store}, manifest_group=manifest_group)
 
 
-@pytest.mark.asyncio
 @requires_obstore
 class TestManifestStore:
-    def test_manifest_store_properties(self, manifest_store):
-        assert manifest_store.read_only
-        assert manifest_store.supports_listing
-        assert not manifest_store.supports_deletes
-        assert not manifest_store.supports_writes
-        assert not manifest_store.supports_partial_writes
-
-    async def test_get_data(self, manifest_store):
-        observed = await manifest_store.get(
-            "foo/c/0.0", prototype=default_buffer_prototype()
-        )
+    def test_manifest_store_properties(self, local_store):
+        assert local_store.read_only
+        assert local_store.supports_listing
+        assert not local_store.supports_deletes
+        assert not local_store.supports_writes
+        assert not local_store.supports_partial_writes
+
+    @pytest.mark.asyncio
+    @pytest.mark.parametrize(
+        "manifest_store",
+        ["local_store", pytest.param("s3_store", marks=pytest.mark.minio)],
```

Comment:
Should the mark be `requires_minio`?

Reply:
They're synonymous, but I will re-add requires_minio since it's more descriptive